packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -2183,6 +2184,11 @@ module.exports = function (log, config, bounces, statsd) {
         ...links,
         uid,
         email,
+        invoiceAmountDue: this._getLocalizedCurrencyString(
```

Comment:
🎉 This is great, it looks like we're getting a localized version of this string!
Is it out of the scope for this patch to also fix $productPaymentCycleOld / $productPaymentCycleNew ?

Reply:
Unfortunately it is out of scope for this patch. We have a [ticket](https://mozilla-hub.atlassian.net/browse/FXA-11662) that includes the improvement mentioned, as well as other updates.